app/views/certificates/index.html.erb

```diff
@@ -5,7 +5,7 @@
       <h1 class="govuk-heading-l">Certificates</h1>
     </div>
       <div class="govuk-inset-text govuk-!-margin-top-0 govuk-!-margin-bottom-9">
-        You need to upload at least one Certificate to enable GovWifi Certificate Based Authentication. This will allow devices you issue (with a corresponding client certificate) to seamlessly authenticate with GovWifi.
+        You need to upload at least one Certificate to enable GovWifi Certificate Based Authentication. This will allow devices you issue (with a corresponding client certificate) to seamlessly authenticate with GovWifi. Certificate must be in .PEM format'
```

Comment:
Is there any associated unit test for this that might need the text updating?

Reply:
There is already a format error message that appears if a .PEM file is not used.

Reply:
it "reports an error" do
      file = Tempfile.new("bad_cert")
      file.write("this is not a certificate")
      file.close

      fill_in "Name", with: "MyCert1"
      attach_file("File", Rails.root.join(file.path))
      click_button "Upload Certificate"

      file.unlink

      expect(page).to have_content "Certificates must be in a .PEM format"
    end

Reply:
Cool, I was just checking if any existing test needed updating, but if all tests are passing, then it's all good.